modules/multi-architecture-import-imagestreams.adoc

```diff
@@ -0,0 +1,72 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/cluster-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="multi-architecture-import-imagestreams_{context}"]
+
+= Importing manifest lists in image streams on your multi-architecture cluster 
+
+On a {product-title} {product-version} multi-architecture cluster, the image streams in the cluster do not import manifest lists automatically. You must manually change the default `importMode` option to the `PreserveOriginal` option in order to import the manifest list.
```

Comment:
I feel like it might be better to lead with the second sentence. I think technically it is passive (I'm not 100% though). What do you think of:
```suggestion
On a {product-title} {product-version} multi-architecture cluster, you must manually change the default `importMode` option to the `PreserveOriginal` option in order to import the manifest list. The image streams in the cluster do not import manifest lists automatically. 
```
Or some version of this.